src/locale/locales/fr/messages.po

```diff
@@ -1532,18 +1547,16 @@ msgstr "Veuillez également entrer votre mot de passe :"
 #: src/view/com/modals/AppealLabel.tsx:72
 #: src/view/com/modals/AppealLabel.tsx:75
 msgid "Please tell us why you think this content warning was incorrectly applied!"
-msgstr ""
+msgstr "Dites-nous pourquoi vous pensez que cet avertissement de contenu a été mal appliqué !"
 
-#: src/view/com/modals/AppealLabel.tsx:72
-#: src/view/com/modals/AppealLabel.tsx:75
 #~ msgid "Please tell us why you think this decision was incorrect."
 #~ msgstr "Dites-nous pourquoi vous pensez que cette décision était incorrecte."
 
-#: src/view/com/composer/Composer.tsx:214
+#: src/view/com/composer/Composer.tsx:215
 msgid "Please wait for your link card to finish loading"
-msgstr ""
+msgstr "Attendez la fin du chargement de votre carte de liaison"
```

Comment:
`carte de liaison` doesn't mean what you think it means (it sounds like an administrative physical card you handle to someone?!). Either use `carte de lien`, or a more liberal translation, like `aperçu de lien` (literally `link preview`).

Reply:
That is great. Thanks, made the change.